### PR TITLE
RDISCROWD-5531 Task Browse Column for "Completed By"

### PR DESCRIPTION
**Pull request by kbecker42**
- Added column for "Completed By" to task browse page.
- _Note, unit test will not pass until themes is updated with new template. See [PR](https://github.com/bloomberg/pybossa-default-theme/pull/381). Updated themes to point to PR branch for unit test passing._

## Screenshots

screenshot

Re: https://github.com/bloomberg/pybossa-default-theme/pull/381

### CODE REVIEW

**Comment by kbecker42** [comment]
_Note, unit test will not pass until themes is updated with new template. See [PR](https://github.com/bloomberg/pybossa-default-theme/pull/381)_

File: pybossa/view/projects.py [hunk 1]
```diff
@@ -1697,6 +1696,28 @@ def respond():
...
+                    user_id = int(user_id)
+                    if not user_info.get(user_id):
+                        # Load from cache.
+                        user_info[user_id] = cached_users.get_user_by_id(user_id).fullname
+                    users.append(user_info[user_id])
+                except ValueError:
+                    # Error locating user id.
+                    current_app.logger.info("User does not have valid user id in get_users_fullname(): %s", user_id)
+            task[result_field] = ", ".join(users)
+        return user_info
```
Comment (XiChenn) [inline pybossa/view/projects.py:1719]: Looks like the return statement is not necessary.

File: pybossa/view/projects.py [hunk 2]
```diff
@@ -1697,6 +1696,28 @@ def respond():
 
         return handle_content_type(data)
 
+    def get_users_fullname(page_tasks, get_users_func, result_field):
```
Comment (XiChenn) [inline pybossa/view/projects.py:1699]: The function name "get_users_fullname" seems misleading. This function modifies each task in the "page_tasks" according to "get_users_func" and it doesn't need to return anything. Shall we give a more meaningful name?

File: pybossa/view/projects.py (hunk 1 above)
Comment (kbecker42) [reply pybossa/view/projects.py:1719]: The function could certainly return None. Returning a value from user_info was intended to make future unit testing efforts easier. If there is a specific preference to return a specific value, we could adjust.

**Pull request merged**